app/uk/gov/hmrc/apisubscriptionfields/repository/SubscriptionFieldsRepository.scala

```diff
@@ -93,26 +93,22 @@ class SubscriptionFieldsMongoRepository @Inject()(mongoDbProvider: MongoDbProvid
     )
   }
 
-  override def fetch(identifier: SubscriptionIdentifier): Future[Option[SubscriptionFields]] = {
-    val selector = selectorForIdentifier(identifier)
+  override def fetch(clientId: ClientId, apiContext: ApiContext, apiVersion: ApiVersion): Future[Option[SubscriptionFields]] = {
+    val selector = selectorForSubscriptionField(clientId.value, apiContext.value, apiVersion.value)
     Logger.debug(s"[fetch] selector: $selector")
     collection.find(selector).one[SubscriptionFields]
   }
 
-  private def selectorForIdentifier(clientId: String, apiContext: String, apiVersion: String): JsObject = {
+  private def selectorForSubscriptionField(clientId: String, apiContext: String, apiVersion: String): JsObject = {
```

Comment:
Could you rename it as `selectorForSubscriptionFields` please?

Reply:
Done